Glossary/Plugin.md

```diff
@@ -0,0 +1,5 @@
+#Plugin
```

Comment:
Spaces are missing after `#` (it is needed for correct rendering)

Reply:
fixed